spec/imsc-hrm.html

```diff
@@ -237,9 +237,154 @@ <h2>Terms and Definitions</h2>
+      </ul>
+
+      <figure id='fig-intro'>
+        <img src="figures/intro-fig-1.svg" alt="The HRM prevents incomplete presentations of IMSC Instance Documents.">
+
+        <figcaption>
+          The HRM allows authors and implementers of <a>presentation
+          processors</a> to agree on the maximum complexity of <a>IMSC Document
+          Instances</a>.
+        </figcaption>
+      </figure>
+
+      <p>As illustrated in <a href="#fig-intro"></a>, by defining a method (the
+      HRM) to compute the complexity of an <a>IMSC Document Instance</a> and
+      specifying a maximum <a>IMSC Document Instance</a> complexity according to
+      that method:</p>
+
+      <ul>
+        <li>authors can ensure that their <a>IMSC Document Instances</a> do not
+        exceed this maximum complexity;</li>
+  
+        <li>implementer can design their presentation processor can render all <a>IMSC
+        Document Instances</a> that do not exceed this maximum complexity.</li>
+      </ul>
+
+    </section>
+  
+    <section>
+      <h3>Why is the HRM needed to limit complexity?</h3>
+
+      <p>The HRM supplements the syntactic and structural constraints imposed in
+      [[IMSC]] by imposing constraints on the contents of the presentation.</p>
+  
+      <p>Because of the temporal and spatial variability of subtitles and
+        captions across types of content, territories and languages, it is not
+        possible to limit the complexity of an <a>IMSC Document Instance</a> using
+        only average values.</p>
+  
+      <p class="example">A constraint of 840 characters per minute would
+        allow a subtitle to contain 840 characters, as long it was the only one
+        within a 60 second window.</p>
+  
+      <p>The HRM achieves a more accurate representation of the complexity of an
+        <a>IMSC Document Instance</a> at any given time by taking into account
+        its past complexity in addition to its instantaneous complexity. The
+        same approach is commonly used in video to limit bitstream complexity,
+        e.g., the Hypothetical Reference Decoder (HRD) specified in
+        [[iso14496-10]].
+      </p>
+    </section>
+  
+    <section>
+      <h3>How does the HRM measures complexity?</h3>
+  
+      <p>
+        The HRM defines an (hypothetical) time required for painting subtitles
+        and captions. Painting includes drawing region backgrounds, rendering
+        and copying text, and decoding and copying images. Complexity is then
+        limited by requiring that the time to paint a subtitle/caption is
+        shorter than the time elapsed since the previous subtitle/caption.
+      </p>
+  
+      <p>The calculation of the time required for painting subtitles and
+      captions is based on a static analysis of the <a>IMSC Document
+      Instances</a> and requires no fetching of external resources or actual
+      rendering. It however assumes that some characters are faster to render
```

Comment:
editorial suggestion:

```suggestion
      rendering. It assumes that some characters are faster to render
```